gobblin-data-management/src/main/java/gobblin/data/management/copy/converter/DecryptConverter.java

```diff
@@ -66,4 +71,12 @@ public String convertSchema(String inputSchema, WorkUnitState workUnit) throws S
     }
   }
 
+  /**
+   * Remove {@value #GPG_EXTENSION} from {@link CopyableFile#getDestination()} and
+   * {@link CopyableFile#getRelativeDestination()}
+   */
+  private void removeExtension(CopyableFile file) {
+    file.setDestination(new Path(file.getDestination().toString().replace(GPG_EXTENSION, "")));
```

Comment:
I looked at this and FileNameUtils.getBaseName as well. They can not be used because a converter needs to remove specific extension. For instance DecryptConverter will only remove decryption extensions (.gpg) and UnGzipConverter will remove only tar.gz from the file name